docs/how_to/page_types.rst

```diff
@@ -0,0 +1,63 @@
+.. versionadded:: 3.0
+
+##########
+Page Types
+##########
+
+We are introducing "Page Types" to make it easier for content editors to create
+pages from predefined presets. This presets contain example plugins and content
+that will be carried over the the designated page while leaving the preset
+untouched.
+
+
+*******************
+Creating Page Types
+*******************
+
+First you need to create a new page through **Add page** from the **Page**
```

Comment:
Only buttons should be **bold**; menu items should be *italic* (fyi, fixing here anyway).